index.d.ts

```diff
@@ -3,36 +3,49 @@
 declare namespace sortPackageJsonExports {
   interface SortPackageJsonFn {
     /**
-      * Sort packageJson object.
-      *
-      * @param packageJson - A packageJson
-      * @param options
-      * @returns Sorted packageJson object
-      */
-    <T extends Record<any, any>>(packageJson: T, options?: Options): T,
+     * Sort packageJson object.
+     *
+     * @param packageJson - A packageJson
+     * @param options
+     * @returns Sorted packageJson object
+     */
+    <T extends Record<any, any>>(packageJson: T, options?: Options): T
 
     /**
-      * Sort packageJson string.
-      *
-      * @param packageJson - A packageJson string.
-      * @param options
-      * @returns Sorted packageJson string.
-      */
-    (packageJson: string, options?: Options): string,
+     * Sort packageJson string.
+     *
+     * @param packageJson - A packageJson string.
+     * @param options
+     * @returns Sorted packageJson string.
+     */
+    (packageJson: string, options?: Options): string
   }
 
-  type ComparatorFunction = (left: string, right: string) => number;
+  type ComparatorFunction = (left: string, right: string) => number
+  function sortObjectBy<T extends Record<any, any>>(
```

Comment:
I wasn't quite sure how to test this, I am not used to hand-writing type definitions

Reply:
added tests